CMakeLists.txt

```diff
@@ -27,6 +27,23 @@
 cmake_minimum_required(VERSION 2.8.11)
 project(opae)
 
+############################################################################
+## Get git hash info, if available #########################################
+############################################################################
```

Comment:
Should we make this a macro?

Reply:
It wasn't one before and is only used once.